test/config-luatest/conditional_section_test.lua

```diff
@@ -0,0 +1,160 @@
+local fun = require('fun')
```

Comment:
I'm going to add one more test case asfter @ImeevMA questions regarding an order of applying of the conditional sections.

```diff
diff --git a/test/config-luatest/conditional_section_test.lua b/test/config-luatest/conditional_section_test.lua
index 99c4c8a091..bb2b098097 100644
--- a/test/config-luatest/conditional_section_test.lua
+++ b/test/config-luatest/conditional_section_test.lua
@@ -78,6 +78,45 @@ g.test_unknown_option = function(g)
     })
 end
 
+-- Verify that the last conditional sections wins if they set the
+-- same option.
+g.test_priority = function(g)
+    helpers.success_case(g, {
+        options = {
+            ['conditional'] = {
+                {
+                    ['if'] = 'tarantool_version > 1.0.0',
+                    process = {
+                        title = 'foo',
+                    },
+                },
+                {
+                    ['if'] = 'tarantool_version > 1.0.0',
+                    process = {
+                        title = 'bar',
+                    },
+                },
+                {
+                    ['if'] = 'tarantool_version > 1.0.0',
+                    process = {
+                        title = 'baz',
+                    },
+                },
+                {
+                    ['if'] = 'tarantool_version < 1.0.0',
+                    process = {
+                        title = 'skipped',
+                    },
+                },
+
+            },
+        },
+        verify = function()
+            t.assert_equals(box.cfg.custom_proc_title, 'baz')
+        end,
+    })
+end
+
 -- Several incorrect configuration cases.
 g.test_validation = function()
     -- 'if' should exists in each conditiona; section.
```

Reply:
Done.